contents/tutorials/beta-feedback.md

```diff
@@ -0,0 +1,135 @@
+'use client'
+import posthog from 'posthog-js'
+import { PostHogProvider } from 'posthog-js/react'
+
+if (typeof window !== 'undefined') {
+  posthog.init('<ph_project_api_key>', {
+    api_host: '<ph_instance_address>',
+    opt_in_site_apps: true
+  })
+}
+
+export function PHProvider({ children }) {
+  return <PostHogProvider client={posthog}>{children}</PostHogProvider>
+}
+```
+
+In `layout.js`, we import the provider and wrap your app in it. 
+
+```js
+// app/layout.js
+import './globals.css'
+import { PHProvider } from './providers'
+
+export default function RootLayout({ children }) {
+  return (
+    <html lang="en">
+      <PHProvider>
+        <body>{children}</body>
+      </PHProvider>
+    </html>
+  )
+}
+```
+
+When we run `npm run dev`, PostHog starts capturing events, the beta is ready to implement, and we can also set up surveys with no additional code. 
+
+## Setting up our beta feature
+
+With PostHog and our early access feature set up, we can set up our beta in our app. This will be a button that goes to a `/beta` page. It conditionally shows based on the `new-page-beta` feature flag for out beta. We can evaluate this flag with the `useFeatureFlagEnabled` React hook. Altogether, a basic component looks like this:
+
+```js
+// app/page.js
+'use client'
+import { useFeatureFlagEnabled } from 'posthog-js/react'
+import Link from 'next/link'
+
+export default function Page() {
+
+  const beta = useFeatureFlagEnabled('new-page-beta')
+
+  return (
+    <div>
+      <h1>Welcome to our homepage</h1>
+      {beta && <Link href="/beta">Go to Beta</Link>}
+    </div>
+  )
+}
+```
+
+We also need a beta page. To add one, create a `beta` folder and a `page.js` file, and create a basic component inside. 
+
+```js
+// app/beta/page.js
+export default function Beta() {
+  
+  return (
+    <div>
+      <h1>Welcome to our beta page</h1>
+      <p>This is very new and cool</p>
+    </div>
+  )
+}
+```
+
+Now, when users opt into the "new beta page" feature, a button shows on the homepage linking to the beta page.
+
+![Opting into beta video](../images/tutorials/beta-feedback/opt-in.mp4)
+
+## Adding a survey to the beta feature page
+
+Lastly, we want to get feedback on our new beta page from the users in our beta. We can add this without adding code using [surveys](/docs/surveys).
+
+To create a survey, go to the [surveys tab](https://app.posthog.com/surveys) in PostHog and click "New survey." Choose the [open feedback template](/templates/in-app-feedback-survey), fill out any details, and open the targeting section. In this section, we target the `new-beta-page` feature flag and the `/beta` URL. This prevents non-beta users from getting the survey. Once done, click save and launch.
+
+![Creating survey video](../images/tutorials/beta-feedback/survey.mp4)
+
+Now when users opt-in to the beta and go to `/beta`, they see a survey asking for their feedback. 
+
+![Survey in-app](../images/tutorials/beta-feedback/survey.png)
+
```

Comment:
Not critical, but I think it needs a closing paragraph. It ended a bit abruptly IMO

Reply:
Wrote one